src/common/tagging/external/tag_group.go

```diff
@@ -246,8 +247,9 @@ func (t *TagGroup) CalculateTagValue(block structure.IBlock, tag Tag) (tags.ITag
 				break
 			}
 		} else if len(gitModifiersCounts) > 1 {
-			// TODO use the CODEOWNERS file to resolve the conflict
-			logger.Info(fmt.Sprintf("Git-modifiers conflict found, fallback to default value %s\n", retTag.Value))
+			if res, found := t.getSectionFromCodeOwners(block); found {
```

Comment:
let's put it behind a flag?

Reply:
it's technically just after:
if blockTagKey == tags.GitModifiersTagKey 

do you think we still need ?

Reply:
IMO a user wouldn't necessary want to involve its `CODEOWNER` file (to avoid unexpected behavior)
Anyhow we need to document this in https://github.com/bridgecrewio/yor/blob/main/docs/3.Custom%20Taggers/Custom_tagger_YAML.md